.github/workflows/integration_tests.yml

```diff
@@ -158,7 +158,7 @@ jobs:
           fetch-depth: 0
           submodules: false
       - name: Use expanded matrix
-        if: github.event.inputs.use_expanded_matrix == '1' || needs.check_trigger.outputs.requested_tests == 'full'
+        if: github.event_name == 'schedule' || github.event.inputs.use_expanded_matrix == '1' || needs.check_trigger.outputs.requested_tests == 'full'
         run: |
           echo "EXPANDED_MATRIX_PARAM=-e=1" >> $GITHUB_ENV
```

Comment:
Could you actually log a ::warning here (similar to what we do for auto-diff mode) so we can be sure it's running on the expanded matrix?

Reply:
log added
https://github.com/firebase/firebase-cpp-sdk/runs/3071396257?check_suite_focus=true#step:3:15